src/steps/intune/steps/devices/__tests__/index.test.ts

```diff
@@ -87,7 +88,10 @@ describe('fetchDevices', () => {
     });
     const context = createMockStepExecutionContext({ instanceConfig: config });
 
-    await fetchAccount(context);
+    await context.jobState.setData(
+      DATA_ACCOUT_TYPE,
```

Comment:
```suggestion
      DATA_ACCOUNT_ENTITY,
```
I suggest this name because and entity is being stored, not a type. Also fixes typo 😁